tests/v0_tests/test_cloud_integration_tests.py

```diff
@@ -27,4 +29,27 @@ def test_add_documents_mock_overwrites_mock_for_cloud_tests(self):
         if marqo_url_before_test:
             self.client_settings["url"] = marqo_url_before_test
         if marqo_cloud_url_before_test:
-            os.environ["MARQO_CLOUD_URL"] = marqo_cloud_url_before_test
\ No newline at end of file
+            os.environ["MARQO_CLOUD_URL"] = marqo_cloud_url_before_test
+
+    def test_index_cleanup_works_with_add_documents_mocked(self):
```

Comment:
add docstring

Reply:
Few questions:
1. what is this test trying to accomplish? 
2. Was this test I recommended?
3. If this was an attempt at the test I recommended, why does this one deviate?

Reply:
The only deviation is that instead of mocking it with side effect of adding docs I just stop mock on the moment of adding documents and then start it again and it tests that cleanup of documents worked anyways.